testing/tests/index.test.js

```diff
@@ -1,13 +1,20 @@
 const fs = require("fs");
 const path = require("path");
+const { execSync } = require("child_process");
 
 const cheerio = require("cheerio");
 
 const buildRoot = path.join("..", "client", "build");
 
-test("content built foo page", () => {
+beforeAll(() => {
+  execSync("node content build --start-clean", {
+    cwd: path.resolve(".."),
```

Comment:
This is just so we run the command from the root rather than from the `testing` workspace.